server/gitrest/packages/gitrest-base/src/routes/summaries.ts

```diff
@@ -116,6 +116,7 @@ async function createSummary(
     repoManagerParams: IRepoManagerParams,
     externalWriterConfig?: IExternalWriterConfig,
     persistLatestFullSummary = false,
+    enableLowIoWrite: "initial" | boolean = false
 ): Promise<IWriteSummaryResponse | IWholeFlatSummary> {
     const lumberjackProperties = {
         ...getLumberjackBasePropertiesFromRepoManagerParams(repoManagerParams),
```

Comment:
Just realized: can we add `enableLowIoWrite` as a property here under `lumberjackProperties`? That way, it will automatically be passed on to `GitWholeSummaryManager` and might make it easier to tell the metric value/datapoints for each case apart

Reply:
Discussed offline that `enableLowIoWrite` is added to metrics internally within `GitWholeSummaryManager` which is the only place that enableLowIoWrite alters logic. Can follow up with a PR to hoist it though